web/packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -78,8 +78,8 @@ export const formatters: Formatters = {
   [eventCodes.ACCESS_REQUEST_REVIEWED]: {
     type: 'access_request.review',
     desc: 'Access Request Reviewed',
-    format: ({ id, reviewer }) =>
-      `User [${reviewer}] reviewed access request [${id}]`,
+    format: ({ id, reviewer, state }) =>
+      `User [${reviewer}] reviewed access request [${id}] and is ${state}`,
```

Comment:
I would just say `User stephen approved access request 12345`.

Reply:
updated to 
<img width="789" alt="image" src="https://github.com/gravitational/teleport/assets/60704961/d0e19a0e-13ac-44b3-ace0-2a84e735f94c">

Reply:
Please see update. I lowercased the `state` variable so it reads in a more friendly manner.